files/u.js/update.json

```diff
@@ -0,0 +1,11 @@
+{
+  "packageManager": "github",
+  "name": "u.js",
+  "repo": "iamso/u.js",
+  "basePath": "dist/",
```

Comment:
basePath should be inside files. Check the examples https://github.com/jsdelivr/libgrabber#examples